components/network/ntp/Makefile

```diff
@@ -26,12 +26,14 @@
 # Copyright (c) 2023, Niklas Poslovski
 #
 
+# for the moment the SHAKE128 digest fails
```

Comment:
I believe this comment does not belong here.  Why do we need it at all?  Isn't the test failure visible in `results-all.master`.  If not, why?

Isn't it possible to skip the test instead?

Reply:
removed